meson.build

```diff
@@ -0,0 +1,55 @@
+project('nnstreamer', 'c', 'cpp',
```

Comment:
Could you append how to use Meson in ./Documentation/ folder?

Reply:
I will update next commit.